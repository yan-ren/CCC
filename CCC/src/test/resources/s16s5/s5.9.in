15 1000000000
010000010000101
