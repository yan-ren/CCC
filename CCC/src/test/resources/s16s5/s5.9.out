010111001111110
